### DISCUSSION TITLE
Support for entire mini.nvim suite

### DISCUSSION THREAD

**Discussion by 231tr0n** [olimorris/onedarkpro.nvim#249] [Ideas]
Hi and thank you for this amazing colorscheme. Is it possible to add support for the entire mini.nvim suite of plugins. It would really help a lot since most of the plugins I use are from mini.nvim

**Comment by olimorris** [comment]
It's most certainly possible 😉. I don't use any of the mini.nvim plugins though so would love to have you PR them. I've outlined how you can do this in the [CONTRIBUTING.md](https://github.com/olimorris/onedarkpro.nvim/blob/main/CONTRIBUTING.md) guide.

I've added a `.lazy.lua` file to the repository which allows you to add and create highlights in realtime and see their impact. It's super quick to style new groups.

**Reply by 231tr0n** [reply]
Cool will try tweaking stuff out and try raising a PR. Also, do you mind if I add more plugins other than mini.nvim namely nvim-treesitter-context, render-markdown.nvim, helpview.nvim and fzf-lua?

**Reply by olimorris** [reply]
Please do, that would be awesome. I suspect render-markdown will be interesting so happy to collaborate on that one.
